examples/BskSim/models/BSK_Fsw.py

```diff
@@ -330,6 +364,46 @@ def SetRWMotorTorque(self):
         messaging.ArrayMotorTorqueMsg_C_addAuthor(self.rwMotorTorqueData.rwMotorTorqueOutMsg, self.cmdRwMotorMsg)
         self.rwMotorTorqueData.rwParamsInMsg.subscribeTo(self.fswRwConfigMsg)
 
+
+    def SetLambertPlannerObject(self, SimBase):
+        """Set the lambert planner object."""
```

Comment:
We usually add the modelTag near the module instantiation higher in the file, any specific reason to having it here?

Reply:
I looked at how C++ modules were configured in the Dynamics file, and this is how it was done there. I did change it now in the FSW file to be consistent.